server/liveblog/blogs/blogs.py

```diff
@@ -147,16 +147,18 @@ def on_created(self, docs):
     def find_one(self, req, **lookup):
         doc = super().find_one(req, **lookup)
         # check if the current user has permission to open a blog
-        if not is_admin(get_user()):
-            # get members ids
-            members = [str(m['user']) for m in doc.get('members', [])]
-            # add owner id to members
-            members.append(doc.get('original_creator'))
-            # check if current user belongs to members, and raise an exeption if not
-            if str(get_user().get('_id')) not in members:
-                roles = get_resource_service('roles').find_one(req=None, _id=get_user().get('role'))
-                if not roles:
-                    raise SuperdeskApiError.forbiddenError(message='you do not have permission to open this blog')
+        user = get_user()
+        if user:
```

Comment:
I guess I couldn't find another way at that moment but now...if I think better, instead of `'blog': Resource.rel('blogs', True)` I should use `'blog': Resource.rel('client_blogs', True)` and it won't be needing a security hack